localization/english/events/event_localization/divine_spark_ritual_events_l_english.yml

```diff
@@ -1,13 +1,13 @@
 ﻿l_english:
-  divine_spark_rituals.001.title:0 "You Have Ascended"
-  divine_spark_rituals.001.desc:0 "Through careful preparations you have become more aligned with the Divine Spark, your Elf Trait has evolved!!"
-  divine_spark_rituals.001.ascend:0 "I have become more! Huzzah!"
+  divine_spark_rituals.001.title:0 "You Have Ascended!"
+  divine_spark_rituals.001.desc:0 "Through careful preparations you have become more aligned with the Divine Spark, and your Elf Trait has evolved!"
```

Comment:
do we need the comma before and?

Reply:
It acts as a pause, without it the sentence looks a bit muddled.